openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -196,9 +196,4 @@ region_name = {{.Values.global.region}}
 [wsgi]
 default_pool_size = {{ .Values.wsgi_default_pool_size | default .Values.global.wsgi_default_pool_size | default 100 }}
 
-[workarounds]
-{{- if .Values.pod.replicas.consoleauth }}
-# This has to be removed when we also remove the deployment of nova-consoleauth
-enable_consoleauth = True
-{{- end }}
 enable_live_migration_to_old_hypervisor = True
```

Comment:
Was `enable_live_migration_to_old_hypervisor` part of the [workarounds] section and is now part of [wsgi]? Or is the empty line meaningful in this syntax (even so, I don't think there was an empty line between `enabled_consoleauth` and `enable_live_migration_to_old_hypervisor`, if I understand `{{- end }}` correctly, i.e. the leading dash makes it as if that tag were on the previous line right after the `True`.

Reply:
Yes, this is indeed a bug.